vpr/src/device/rr_graph_builder.h

```diff
@@ -65,6 +65,10 @@ class RRGraphBuilder {
         node_storage_.set_node_coordinates(id, x1, y1, x2, y2);
     }
 
+    /** @brief Set the node direction */
```

Comment:
Need to add more comments about this API. 
``The node direction is only available of routing channel nodes, such as x-direction routing tracks (CHANX)  and y-direction routing tracks (CHANY)``.

You can refer to the comments at https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/a86083faf52f9b98df80f0840267c9a5c757ba95/vpr/src/route/rr_graph_storage.h#L44